recipes/renode-cli/build.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+# Consolidated script logic
+main() {
```

Comment:
I'm not sure this script needs separate functions, as we're not iterating over these functions or passing special parameters.  We can just run everything sequentially.

Reply:
Done